src/QMCTools/LCAOHDFParser.cpp

```diff
@@ -164,6 +164,7 @@ void LCAOHDFParser::parse(const std::string& fname)
     {
       hin.read(ci_size, "NbDet");
       hin.read(ci_nstates, "nstate");
+      hin.read(nbexcitedstates, "nexcitedstate");
```

Comment:
If the h5 stores ground, 1st, 2nd CI coefficients in one h5, what does this entry mean?

Reply:
This entry is to specify how many excited coefficients to expect.

Reply:
If the number is 5, does it mean there are 5 excited states to be expected? Including ground or not? Please add a comment line above the declaration of nbexcitedstates in the header file.